src/index.html

```diff
@@ -1,13 +1,473 @@
+                    grid__item
+                    grid__item--1-1
+                    grid__item--desktop-1-6"
+                  >
+                    <div class="place__photo-wraper">
+                      <img
+                        class="place__photo"
+                        src="./images/photos/3.jpg"
+                        alt="Faena House / Miami Beach, FL 33140 ">
+                    </div>
+                  </div>
+
+                  <div class="
+                    grid__item
+                    grid__item--2-2
+                    grid__item--desktop-7-12"
+                  >
+                    <div class="place__photo-wraper">
+                      <img
+                        class="place__photo"
+                        src="./images/photos/4.jpg"
+                        alt="Miami Beach, FL 33140">
+                    </div>
+                  </div>
+                </div>
+              </div>
+
+              <div class="place__desciption">
+                <a
+                  class="place__link"
+                  href="https://goo.gl/maps/ZXLv7Usa7WzHcg3E8"
+                  target="_blank"
+                >
+                  Edition Residens
+                </a>
+                / Miami Beach, FL 33139 
+              </div>
+            </div>
+
+            <div class="place">
+              <div class="place_photos">
+                <div class="grid">
+                  <div class="
+                    grid__item
+                    grid__item--1-1
+                    grid__item--desktop-1-4"
+                  >
+                    <div class="place__photo-wraper">
+                      <img
+                        class="place__photo"
+                        src="./images/photos/5.jpg"
+                        alt="Marina Palms / North Miami Beach, FL 33162">
+                    </div>
+                  </div>
+
+                  <div class="
+                    grid__item
+                    grid__item--2-2
+                    grid__item--desktop-5-12"
+                  >
+                    <div class="place__photo-wraper">
+                      <img
+                        class="place__photo"
+                        src="./images/photos/6.jpg"
+                        alt="North Miami Beach, FL 33162">
+                    </div>
+                  </div>
+                </div>
+              </div>
+
+              <div class="place__desciption">
+                <a
+                  class="place__link"
+                  href="https://goo.gl/maps/ZXLv7Usa7WzHcg3E8"
+                  target="_blank"
+                >
+                  Faena House
+                </a>
+                / Miami Beach, FL 33140
+              </div>
+            </div>
+
+            <div class="grid grid--desktop">
+              <div class="grid__item grid__item--6-7">
+                <a href="#gallery" class="button">
+                  View all
+                </a>
```

Comment:
Font weight of buttons should be bold https://prnt.sc/1tx8tcs, but it's normal on your page https://prnt.sc/1tx8pr7, make sure to fix it for all buttons